### ISSUE TITLE
Populate accessibility statement

### ISSUE THREAD

**Issue by samuelscandrett** [k-int/arches-lincoln#24]
Arches for HERs comes pre-loaded with an accessibility statement plugin (with the UUID 8688eaf9-3605-4384-823c-d01c0d210f82), however it does not have content. For Arcade, we need to populate it with text provided by the City of Lincoln council.

This could be done with a project migration
